fetch.bs

```diff
@@ -4683,6 +4705,7 @@ dictionary RequestInit {
   DOMString integrity;
   boolean keepalive;
   any window; // can only be set to null
+  CancelationSignal signal;
```

Comment:
This is `CancelationSignal` rather than `FetchSignal`, as it'd be nice to allow the generic signal here. I'm guessing I'll be able to do something like "If signal is a `FetchSignal`…" later to handle things like priority changes.